src/state/_common/selectors.js

```diff
@@ -471,13 +472,22 @@ const getIndexTotal = (getSubstate) => {
 	);
 };
 
+/**
+ * 
+ * @param {func} getSubstate 
+ * @return {Array}
+ */
 const getIndexesByFilteredItem = (getSubstate) => {
 	return createSelector([
 		getIndexes(getSubstate),
 		(state, item) => item,
 		],
 		(indexes, item) => {
-			return indexes.filter((index) => commonHelpers.itemFitFilter(index.filter, item));
+			if(!_.isEmpty(indexes)){
+				return indexes.filter((index) => commonHelpers.itemFitFilter(index.filter, item));
+			} else {
+				return [];
+			}
```

Comment:
Return null for consistency & easier testing
```suggestion
			return indexes && indexes.filter((index) => commonHelpers.itemFitFilter(index.filter, item)) || null;
```